lib/workflows/load.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const dir = require('node-dir');
+const yaml = require('js-yaml');
+const kebab = require('lodash/kebabCase');
```

Comment:
As we discussed in API, we should use `underscore.string` instead of lodash's `kebabCase`